### PR TITLE
test: initial setup for integration tests

### PR DESCRIPTION
**Pull request by cjcobb23**
Code to instantiate all of the contracts in a simulation environment. Can execute contract calls and the contracts will talk to each other. No need for mock contracts.

### CODE REVIEW

File: integration-tests/src/test.rs [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+    use axelar_wasm_std::nonempty;
+    use connection_router::state::ChainName;
+    use cosmwasm_std::{coins, Addr, Binary, Deps, Env, HexBinary, StdResult, Uint128, Uint256};
+    use cw_multi_test::{App, ContractWrapper, Executor};
+
+    use multisig::key::PublicKey;
+
+    const AXL_DENOMINATION: &str = "uaxl";
+    #[test]
+    fn test() {
```
Comment (fish-sammy) [inline integration-tests/src/test.rs:13]: This is a bad name.

**Review by fish-sammy — CHANGES_REQUESTED** [review]

File: integration-tests/src/test.rs [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+        }
+    }
+
+    // return the all-zero array with the first bytes set to the bytes of `index`
+    fn generate_key(seed: usize) -> KeyPair {
+        let index_bytes = seed.to_be_bytes();
+        let mut result = [0; 64];
+        for (i, &b) in index_bytes.iter().enumerate() {
+            result[i] = b;
+        }
```
Comment (cgorenflo) [inline integration-tests/src/test.rs:142]: is there no direct way to set this up without enumeration?

**Review by cgorenflo — APPROVED** [review]
looks good apart from the open comments

File: integration-tests/src/test.rs (hunk 1 above)
Comment (cjcobb23) [reply integration-tests/src/test.rs:14]: lol this is just a placeholder to show how the setup of the testing environment works. It's not really a test. But I can change it to something a little better for now anyways.

**Review by fish-sammy — APPROVED** [review]

**Pull request merged**